rustup-init.sh

```diff
@@ -376,9 +376,9 @@ downloader() {
     if [ "$1" = --check ]; then
         need_cmd "$_dld"
     elif [ "$_dld" = curl ]; then
-        curl -sSfL "$1" -o "$2"
+        curl --proto =https --tlsv1.2 --silent --show-error --fail --location "$1" --output "$2"
```

Comment:
It looks like this may be [causing issues on Travis](https://travis-ci.com/rust-lang/cargo/jobs/195079192)?

Reply:
We can't tell why without debug logging. We don't know whether it is solely because of this change or due to something else, such as the state of the infra.

Reply:
This has been resolved as safely as we know how with 1.18.1

Reply:
@kinnison: Can you please explain a bit more?